index.js

```
@@ -19,11 +19,14 @@ module.exports = () => {
 		};
 	}).catch(() => {
 		return pify(getos).then(res => {
+			if (!res) {
+				res = { };
```

I rather throw an error.